src/docs/release/breaking-changes/network-policy-ios-android.md

```diff
@@ -60,14 +60,37 @@ your manifest would contain the following:
 </application>
 ```
 
-Furthermore:
+### Allowing cleartext connection for debug builds
+
+If you would like to allow HTTP connections for Android debug
+builds, you can add the following snippet to your $project_path\android\app\src\debug\AndroidManifest.xml:
+
+```
+<application android:usesCleartextTraffic="true"/>
+```
+
+For iOS, you can follow [these instructions](https://flutter.dev/docs/development/add-to-app/ios/project-setup#local-network-privacy-permissions) to create a `Info-debug.plist` and put this in:
+
+```
+<key>NSAppTransportSecurity</key>
+<dict>
+    <key>NSAllowsArbitraryLoads</key>
+    <true/>
+</dict>
+```
+
+We **do not** recommend you do this for your release builds.
+
+## Additional Information
 
 * Build time configuration is the only way to change
   network policy. It cannot be modified at runtime.
 * Localhost connections are always allowed.
 * You can allow insecure connections only to domains.
   Specific IP addresses are not accepted as input.
-  This is in line with what platforms support.
+  This is in line with what platforms support. If you would
+  like to be able to whitelist IP addresses the only option
```

Comment:
For inclusion, we no longer use "whitelist" or "blacklist". Please rephrase. For example:

```
If you would like to _allow_ IP addresses the only option...
```

https://developers.google.com/style/word-list#blacklist